relayer/relayer_manager.go

```diff
@@ -31,12 +34,21 @@ func (r *Relayer) SetLogger(logger *log.Logger) {
 }
 
 func (r *Relayer) GetRelayerStatus(config.RollappConfig) string {
-	if r.ChannelReady() {
-		return fmt.Sprintf("Active src, %s <-> %s, dst", r.SrcChannel, r.DstChannel)
+	bytes, err := os.ReadFile(r.statusFilePath())
```

Comment:
I think u should leave this check `if r.ChannelReady() {`
no need to poll forever the file once the channel is active